.gitignore

```diff
@@ -0,0 +1,29 @@
+# dependencies
+node_modules
+
+# Development logs
+logs/
+
+# dotenv environment variables file
+.env
+
+# Mac files
+.DS_Store
+
+# Windows files
+Thumbs.db
+
+# Production build
+build/
+dist/
+
+# IDE and editor files
+.idea/
+.vscode/
+
+# Test coverage reports
+coverage/
+
+# Miscellaneous
+*.log
+*.env
```

Comment:
**suggestion:** Same changes as `backend/.gitignore`

Reply:
Are you saying change in frontend .gitignore to match ?